lib/wal/sources/Exception/WalError.cpp

```diff
@@ -0,0 +1,20 @@
+//
+// Created by Zoe Roux on 2021-05-14.
+//
+
+#include "WalError.hpp"
+
+namespace WAL
```

Comment:
If ctors are only for calling parents ctors, I'd like to put them in the header file

Reply:
Why?